src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const url = `
+   https://mate-academy.github.io/phone-catalogue-static/api/phones.json`;
+const detailsUrl = `
+  https://mate-academy.github.io/phone-catalogue-static/api/phones/`;
+const getPhones = () => {
+  return fetch(url)
+    .then((res) => res.json())
+    .catch((reject) => {
+      setTimeout(() => {
+        return new Error(reject);
+      }, 5000);
+    });
+};
+const getPhonesDetails = (ids) => {
+  fetch(`${detailsUrl}` + `${ids}.json`)
```

Comment:
```suggestion
const getPhonesDetails = (id) => {
  fetch(`${detailsUrl}` + `${id}.json`)
```
Ты же один id сюда передаешь